### PR TITLE
Remove homu test from README

### PR DESCRIPTION
**Pull request by ConnorGBrewster**
See title

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,5 @@ Salt
 ===
 [![Build Status](https://travis-ci.org/jspauldo98/Salt.svg?branch=master)](https://travis-ci.org/jspauldo98/Salt?branch=master)
 
-Homu Test
```
Comment (jspauldo98) [inline README.md]: that's questionable

**Comment by jspauldo98** [comment]
@homu-salt r=nspauld1

**Comment by homu-salt** [comment]
:pushpin: Commit ab3bd52 has been approved by `nspauld1`

**Comment by homu-salt** [comment]
:hourglass: Testing commit ab3bd52 with merge 356092d...

**Comment by homu-salt** [comment]
:sunny: Test successful - [travis](https://travis-ci.org/jspauldo98/Salt/builds/128731437)

File: README.md (hunk 1 above)
Comment (ConnorGBrewster) [inline README.md]: NO IT ISN'T SHUT UP

**Pull request merged**